<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadedGameWindow</class>
 <widget class="QMainWindow" name="LoadedGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>500</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartNewGame">
    <property name="geometry">
     <rect>
      <x>689</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start new game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadGame">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Load game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
